src/applications/vaos/tests/e2e/va-appointment.cypress.spec.js

```diff
@@ -283,7 +283,7 @@ describe('VAOS direct schedule flow', () => {
     newApptTests.confirmationPageTest(additionalInfo);
   });
 });
-describe('VAOS direct schedule flow with a Cerner site', () => {
+describe.skip('VAOS direct schedule flow with a Cerner site', () => {
```

Comment:
Skipping tests instead of removing them, because it bloats the PR. I'll remove them in a follow up PR.